schema/settings.json

```diff
@@ -14,6 +14,12 @@
       "title": "Highlight",
       "description": "Highlight the execution time on completion",
       "default": true
+    },
+    "display": {
+      "type": "string",
```

Comment:
I believe that enums are supported. For example check out [core](https://github.com/jupyterlab/jupyterlab/blob/0c609d6a83f9a0e1d54632e006fdcb78fde4c041/packages/notebook-extension/schema/tracker.json#L305)

Reply:
I've added the enum-field.